e2e/features/save-application.feature

```diff
@@ -87,11 +83,7 @@ Feature: Complete the appeal application online
     And Within the Address fieldset, I should see United Kingdom for the Country field
     And I should see My client's protection claim was refused for the Appeal reason field
     And I should see Yes for the Are there any new reasons your client wishes to remain in the UK or any new grounds on which they should be permitted to stay? field
-    And I should see Birth of a child for the New matters field
-    And I should see New relationship for the New matters field
-    And I should see Removal would disrupt family life for the New matters field
-    And I should see Appellant now claims to be a refugee for the New matters field
-    And I should see New marriage for the New matters field
+    And I should see Birth of a child for the Explain what the new matters are and why they are relevant to this appeal. field
```

Comment:
I was thinking this before, it might be useful at some point to introduce a way to separate values / inputs from the cucumber step definition code, e.g. using quotes, brackets, something else, as this would probably be easier to read:
`And I should see "Birth of a child" for the "Explain what the new matters are and why they are relevant to this appeal." field`

Reply:
heard from @fantasticjamieburns that this is covered in a separate PR 👍